src/qt/CMakeLists.txt

```diff
@@ -85,6 +85,9 @@ target_include_directories(bitcoinqt
   PUBLIC
     $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/src>
 )
+set_property(SOURCE macnotificationhandler.mm
+  APPEND PROPERTY COMPILE_OPTIONS -Wno-error=deprecated-declarations
```

Comment:
This isn't in the master branch? Also not sure why it's -Wno-error? We aren't moving to a state where we are using, or trying to enable `-Werror` by default?

Reply:
> This isn't in the master branch?

https://github.com/bitcoin/bitcoin/pull/29362 has been submitted to the master branch.

Reply:
Ok, dropped.